to.etc.domui/src/main/java/to/etc/domui/server/AppFilter.java

```diff
@@ -73,6 +73,9 @@
 
 	static private String m_appContext;
 
+	@Nullable
+	static private Boolean m_isProductionMode;
```

Comment:
This should not be a Boolean but a boolean, and it is the wrong way around; it should be isTestMode (as that is what it is called everywhere) and it should default to false.

And boolean field names never have "is" in their name. It would create odd setter names.